samples/actions/terraform-plan.yaml

```diff
@@ -0,0 +1,101 @@
+## Cloning the Repository
+- action: 
+  uses: git-clone
+  with:
+    branch: main
+    repo-name: Product-Management
+    connector: cd-demo
+
+
+## Terraform Plan 
+- action:
+    uses: terraform-plan
+    with:
+      command: apply | destroy
+      aws-provider: account.aws_connector
+      workspace: dev
+      backendConfig: |-
```

Comment:
`backend-config` instead of `backendConfig`.  or maybe we could just call it `config`?  any need to prefix with `backend`?

Reply:
The terraform calls that field is called backend configuration, we should keep backend because its similar to terraform naming.